lua/autorun/webaudio.lua

```diff
@@ -474,7 +474,7 @@ if SERVER then
 			net.Broadcast()
 		end
 	end
-	WebAudioStatic.sendCustomWhitelist = sendCustomWhitelist
+	WebAudio.sendCustomWhitelist = sendCustomWhitelist
```

Comment:
This one should still be WebAudioStatic tho, we just use WebAudio.(*) for the getters to know that it's a static global function and to work with autorefresh (Although that doesn't apply in this file)

Reply:
Ahh didn't see `__index = WebAudioStatic,`